gobblin-utility/src/main/java/org/apache/gobblin/util/jdbc/DataSourceProvider.java

```diff
@@ -37,6 +38,7 @@
 /**
  * A provider class for {@link javax.sql.DataSource}s.
  */
+@Slf4j
```

Comment:
curious, do we need this in addition to the `LoggerFactory.getLogger` call?

Reply:
good catch, I didn't notice the existing LOGGER.